packages/wp-build/src/package-utils.mjs

```diff
@@ -46,19 +47,11 @@ export function getPackageInfo( fullPackageName ) {
 		return packageJsonCache.get( fullPackageName );
 	}
 
-	try {
-		const resolved = import.meta.resolve(
-			`${ fullPackageName }/package.json`
-		);
-		const packageJson = JSON.parse(
-			readFileSync( fileURLToPath( resolved ), 'utf8' )
-		);
-		packageJsonCache.set( packageJson.name, packageJson );
-		return packageJson;
-	} catch ( error ) {
-		packageJsonCache.set( fullPackageName, null );
-		return null;
-	}
+	const resolved = import.meta.resolve( `${ fullPackageName }/package.json` );
+	const result = getPackageInfoFromFile( fileURLToPath( resolved ) );
+	packageJsonCache.set( fullPackageName, result );
```

Comment:
This cache kinda feels unnecessary? What's the bottleneck if we don't have this cache? `import.meta.resolve` ?

Reply:
yes, my assumption is that node resolution is probably a bit expensive.